relx.config

```diff
@@ -25,3 +25,7 @@
 {sys_config, "priv/app.config"}.
 {include_src, false}.
 {vm_args, "./priv/vm.args"}.
+{overlay, [
+           {mkdir, "ebin"},
```

Comment:
`ebin/` directory name is misleading, could you please change it to more meaningful?